conduit-spotify/src/Concerns/ProvidesLibraryOverview.php

```
@@ -0,0 +1,118 @@
+namespace Conduit\Spotify\Concerns;
+
+use Conduit\Spotify\Contracts\ApiInterface;
+
+trait ProvidesLibraryOverview
+{
+    public function getLibraryOverview(ApiInterface $api): array
+    {
+        $playlists = $this->getMemoizedPlaylists($api);
+        $totalTracks = 0;
+        $totalDuration = 0;
+        $uniqueArtists = [];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$uniqueArtists'. (Unused Code Rules)

(UnusedLocalVariable)